frontend/src/scenes/settings/organization/InviteModal.tsx

```diff
@@ -221,10 +223,53 @@ export function InviteModal({ isOpen, onClose }: { isOpen: boolean; onClose: ()
     const { user } = useValues(userLogic)
     const { preflight } = useValues(preflightLogic)
     const { invitesToSend, canSubmit } = useValues(inviteLogic)
-    const { resetInviteRows, inviteTeamMembers } = useActions(inviteLogic)
+    const { message } = useValues(inviteLogic)
+    const { stepKey, productKey } = useValues(onboardingLogic)
+    const { isUserInNonTechnicalTest } = useValues(sdksLogic)
+    const { resetInviteRows, inviteTeamMembers, updateMessage } = useActions(inviteLogic)
+
+    const inviteContext = (userMessage: string): string => {
+        return [
+            userMessage,
+            '',
+            '--- Setup Context ---',
+            `This message is auto-generated. Your team needs to complete the ${stepKey} step for ${productKey}`,
+            'Please ensure the installation is completed for everything to run smoothly.',
+            '',
+        ].join('\n')
+    }
 
     const validInvitesCount = invitesToSend.filter((invite) => invite.isValid && invite.target_email).length
 
+    const getModalDescription = (): JSX.Element => {
+        if (!preflight?.email_service_available) {
+            return (
+                <p>
+                    This PostHog instance isn't configured to send emails. In the meantime, you can generate a link for
+                    each team member you want to invite. You can always invite others at a later time.{' '}
+                    <strong>Make sure you share links with the organization members you want to invite.</strong>
+                </p>
+            )
+        }
+
+        if (isUserInNonTechnicalTest) {
+            return (
```

Comment:
This part should still include the `An invite is specific to an email address and expires after 3 days.` message.